### PR TITLE
Add blog post about Openstack-Ansible inventory

### PR DESCRIPTION
**Pull request by nrb**
In a comment on the pull request, include the following information and check off the list.

 - [x] Mini bio
  - Soft capped to 75 words
 - [x] Desired date of publishing
  - Make sure this matches the `date` in the front matter
 - [x] Social media contact info
 - [x] Redaction: (yes|no)
  - Allow redaction, editing or structure changes by technical writer staff
 - [x] Comments: (yes|no)
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by nrb** [comment]
# Mini bio
 Nolan Brubaker has been a software developer with the Rackspace Public Cloud group for 2.5 years.
 # Desired date of publishing
  2017-02-28
 # Social media contact info
     twitter: @palendae
 # Redaction:
  yes
 # Comments:
 no

File: _posts/2017-02-28-osa-inventory-happenings.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "OpenStack-Ansible Inventory Happenings"
+date: 2017-02-28 23:59
+comments: false
+author: Nolan Brubaker
+published: true
+categories:
+    - OpenStack-Ansible
```
Comment (DonNateR) [inline _posts/2017-02-28-osa-inventory-happenings.md]: Delete `Openstack-Ansible` as a category. The build error is caused by a rule that states a category has to be used by at least seven different posts.

**Comment by DonNateR** [comment]
@nrb: See my review for how to fix the build error.

File: _posts/2017-02-28-osa-inventory-happenings.md (hunk 1 above)
Comment (nrb) [inline _posts/2017-02-28-osa-inventory-happenings.md]: Thanks, will fix this.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-af6912aede/).

File: _posts/2017-02-28-osa-inventory-happenings.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+to deploying and maintaining an Ansible-based cluster.
+
+This system has undergone several changes since the initial Icehouse
+release. Learn what's happened since and what's in store for the future.
+
+<!-- more -->
+
+# Background
+
+The OpenStaack-Ansible project was first introduced with the Icehouse
```
Comment (cyrichardson) [inline _posts/2017-02-28-osa-inventory-happenings.md]: OpenStack

**Comment by DonNateR** [comment]
@nrb: You mentioned wanted to build the blog locally earlier. If you want to see your formatting and how the blog will look without building locally, just use this staging link: https://staging.developer.rackspace.com/build-af6912aede/blog/osa-inventory-happenings/

**Comment by nrb** [comment]
@DonNateR Thanks! I did get around my issues locally, though I'm missing CSS there. I'll just use the staging environment.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-aeb1e83d98/)

File: _posts/2017-02-28-osa-inventory-happenings.md (hunk 2 above)
Comment (nrb) [inline _posts/2017-02-28-osa-inventory-happenings.md]: Done, along with some other clean up on that paragraph.

**Comment by catlook** [comment]
Let me know when you've applied the edits you want to, and I'll schedule for publish on the 28th.

**Comment by nrb** [comment]
@catlook Thanks! I thought of a couple other things to add, but I don't think it's going to be too drastically different.

**Comment by catlook** [comment]
Sure…ping me when it’s ready ☺

-- Cat Lookabaugh, Sr. Information Developer
Rackspace Hosting – the #1 Managed Cloud Company
Info Dev Team      cat.lookabaugh@rackspace.com
Office: (210) 312-8193   Cell: (210) 364-1060

**Comment by catlook** [comment]
Ready to publish tomorrow?

-- Cat Lookabaugh, Sr. Information Developer
Rackspace Hosting – the #1 Managed Cloud Company
Info Dev Team      cat.lookabaugh@rackspace.com
Office: (210) 312-8193   Cell: (210) 364-1060

**Comment by nrb** [comment]
@catlook I actually didn't get to write that content over the weekend, but I also don't want to make a bunch of changes today given the tight timeline.

So yes, I'm comfortable with it being published as is. Thanks!

**Comment by catlook** [comment]
No problem.  You can revise after we publish tomorrow ☺

-- Cat Lookabaugh, Sr. Information Developer
Rackspace Hosting – the #1 Managed Cloud Company
Info Dev Team      cat.lookabaugh@rackspace.com
Office: (210) 312-8193   Cell: (210) 364-1060

**Comment by catlook** [comment]
And it is published!  https://developer.rackspace.com/blog/osa-inventory-happenings/

-- Cat Lookabaugh, Sr. Information Developer
Rackspace Hosting – the #1 Managed Cloud Company
Info Dev Team      cat.lookabaugh@rackspace.com
Office: (210) 312-8193   Cell: (210) 364-1060


From: Cat Lookabaugh <cat.lookabaugh@RACKSPACE.COM>
Date: Monday, February 27, 2017 at 9:42 AM
To: rackerlabs/docs-developer-blog <reply@reply.github.com>
Cc: Cat Lookabaugh <cat.lookabaugh@RACKSPACE.COM>
Subject: Re: [rackerlabs/docs-developer-blog] Add blog post about Openstack-Ansible inventory (#244)

No problem.  You can revise after we publish tomorrow ☺

-- Cat Lookabaugh, Sr. Information Developer
Rackspace Hosting – the #1 Managed Cloud Company
Info Dev Team      cat.lookabaugh@rackspace.com
Office: (210) 312-8193   Cell: (210) 364-1060

**Comment by nrb** [comment]
@catlook What's the protocol on reposting such articles to our own blogs? Allowed, or no?

**Pull request merged**